### PR TITLE
Add send_signed_change_request management command.

### PR DESCRIPTION
**Pull request by olegtropinin**

### CODE REVIEW

File: node/blockchain/utils/lock.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 
 def get_pymongo_client():
-    if (client := thread_storage.pymongo_client) is None:
+    if (client := getattr(thread_storage, 'pymongo_client', None)) is None:
```
Comment (dmugtasimov) [inline node/blockchain/utils/lock.py:15]: why?

File: node/blockchain/management/commands/send_signed_change_request.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        )
+        coin_transfer_parser.add_argument('--amount', type=int, required=True, help='Amount of coins')
+        coin_transfer_parser.add_argument('--is_fee', type=bool, default=None, help='Is fee')
+        coin_transfer_parser.add_argument('--memo', type=bool, default=None, help='Transaction message')
+
+    def handle(self, address, signing_key, scr_type, *args, **options):
+        scr_message_class = SIGNED_CHANGE_REQUEST_MESSAGE_MAP[scr_type]
+
+        blockchain_facade = BlockchainFacade.get_instance()
+        scr_arguments = dict(account_lock=blockchain_facade.get_account_lock(derive_public_key(signing_key)))
```
Comment (dmugtasimov) [inline node/blockchain/management/commands/send_signed_change_request.py:55]: this is not correct, account lock must be take from remote blockchain, not local

File: node/blockchain/management/commands/send_signed_change_request.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        # Coin Transfer
+        coin_transfer_parser = subparsers.add_parser(
+            Type.COIN_TRANSFER.name, help='Coin transfer signed change request'
+        )
+        coin_transfer_parser.add_argument(
+            '--recipient', type=AccountNumber, required=True, help='Account number of recipient'
+        )
+        coin_transfer_parser.add_argument('--amount', type=int, required=True, help='Amount of coins')
+        coin_transfer_parser.add_argument('--is_fee', type=bool, default=None, help='Is fee')
```
Comment (dmugtasimov) [inline node/blockchain/management/commands/send_signed_change_request.py:48]: --is-fee

not need for type and default

File: node/blockchain/management/commands/send_signed_change_request.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            Type.NODE_DECLARATION.name, help='Node declaration signed change request'
+        )
+        node_declaration_parser.add_argument('--identifier', type=AccountNumber, required=True, help='Node identifier')
+        node_declaration_parser.add_argument(
+            '--addresses', nargs='+', type=str, required=True, help='One of many network addresses'
+        )
+        node_declaration_parser.add_argument('--fee', type=int, required=True, help='Fee amount')
+
+        # Coin Transfer
+        coin_transfer_parser = subparsers.add_parser(
```
Comment (dmugtasimov) [inline node/blockchain/management/commands/send_signed_change_request.py:41]: multiple transactions should be allowed here

File: node/blockchain/utils/lock.py (hunk 1 above)
Comment (olegtropinin) [reply node/blockchain/utils/lock.py:15]: I got a case when class "thread_storage" doesn't have "pymongo_client" attribute.

File: node/blockchain/management/commands/send_signed_change_request.py (hunk 2 above)
Comment (olegtropinin) [reply node/blockchain/management/commands/send_signed_change_request.py:55]: Ok, I will fix it.

File: node/blockchain/utils/lock.py (hunk 1 above)
Comment (dmugtasimov) [reply node/blockchain/utils/lock.py:15]: you should remove `thread_storage.pymongo_client = None` above then

Comment (olegtropinin) [reply node/blockchain/utils/lock.py:15]: I removed it. Thank you.

File: node/blockchain/management/commands/send_signed_change_request.py (hunk 3 above)
Comment (dmugtasimov) [reply node/blockchain/management/commands/send_signed_change_request.py:48]: action='store_true'

**Comment by olegtropinin** [comment]
Could you review my comments?

File: node/blockchain/management/commands/send_signed_change_request.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+        subparsers = parser.add_subparsers(title='subcommands', dest='scr_type', required=True)
+
+        # Node Declaration
+        node_declaration_parser = subparsers.add_parser(
+            Type.NODE_DECLARATION.name, help='Node declaration signed change request'
+        )
+        node_declaration_parser.add_argument('--identifier', type=AccountNumber, required=True, help='Node identifier')
+        node_declaration_parser.add_argument(
+            '--addresses', nargs='+', type=str, required=True, help='One of many network addresses'
```
Comment (dmugtasimov) [inline node/blockchain/management/commands/send_signed_change_request.py:36]: `--address`

**Pull request merged**